src/PrimeTower.cpp

```diff
@@ -139,6 +141,17 @@ void PrimeTower::generatePaths_denseInfill(const SliceDataStorage& storage)
 }
 
 
+void PrimeTower::generateStartLocations(const SliceDataStorage& storage)
+{
+    PolygonsPointIndex segment_start; // from where to start the sequence of wipe points
+    PolygonsPointIndex segment_end; // where to end the sequence of wipe points
+
+    segment_start = segment_end = PolygonUtils::findNearestVert(middle, outer_poly);
```

Comment:
Two things about this:
* If we're spreading out the points anyway, it doesn't matter that it starts with any particular vertex.
* Isn't the middle equally far away from all vertices? I think the result of this will be random based on rounding errors or the bias in the `findNearestVert` algorithm.

It's easier to start at vertex 0.

Reply:
Done in 13c94392. Also updated the function's documentation.